src/DnsResolver/README.md

```diff
@@ -73,9 +73,9 @@ In this directory, run AutoRest:
 require:
```

Comment:
please specify the latest commit of swagger spec as branch:
```
branch: ${commit}
```

Reply:
Added the commit.